pkgs/data/fonts/inter/default.nix

```diff
@@ -2,19 +2,19 @@
 
 stdenvNoCC.mkDerivation rec {
   pname = "inter";
-  version = "3.19";
+  version = "4.0";
 
   src = fetchzip {
     url = "https://github.com/rsms/inter/releases/download/v${version}/Inter-${version}.zip";
     stripRoot = false;
-    hash = "sha256-6kUQUTFtxiJEU6sYC6HzMwm1H4wvaKIoxoY3F6GJJa8=";
+    hash = "sha256-hFK7xFJt69n+98+juWgMvt+zeB9nDkc8nsR8vohrFIc=";
   };
 
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out/share/fonts/opentype
-    cp */*.otf $out/share/fonts/opentype
+    mkdir -p $out/share/fonts/truetype
+    cp Inter.ttc InterVariable*.ttf $out/share/fonts/truetype
```

Comment:
I see that you've switched to the variable font now. Should we include the non-variable otfs as a fallback too? I don't know much about the font stack but perhaps not all applications support the variable fonts?

Reply:
Both variable and non-variable fonts are packaged in our new version. And as I have mentioned this in the PR description:

> Note that Inter Variable uses “Inter Variable” family name instead of sharing “Inter” family name, including them would not cause conflicts.

So applications using “Inter” will still use the non-variable version. Only when the name “Inter Variable” is specified, the variable font will take place.

Reply:
Oh cool! I didn't know about the ttc format until now. Thanks!